README.md

```diff
@@ -229,6 +230,14 @@ await service.removeCoupon('COUPON');
 await service.removeGiftCertificate('GIFT');
 ```
 
+### Initialize spam protection
+
+You can also enable spam protection to prevent bots and other types of automated abuse in your checkout.
```

Comment:
cc @bc-charlesho with regards to wording

Reply:
Yes I reviewed the wording last week with Robin. We took ideas from the copy in the corresponding KB article for reCAPTCHA.